content/docs/command-reference/config.md

```diff
@@ -179,11 +179,11 @@ This section contains the following options, which affect the project's
   > set.
 
 - `cache.shared` - permissions for newly created or downloaded cache files and
-  directories. The default is `0o664`(rw-r--r--) for files and `0o755`
-  (rwxr-xr-x) for directories. The only accepted value right now is `group`,
-  which makes DVC use `0o664` (rw-rw-r--) for files and `0o775` (rwxrwxr-x) for
-  directories, which is useful when you are using a a
-  [shared development server](/doc/use-cases/shared-development-server).
+  directories. The only accepted value right now is `group`, which makes DVC use
+  `664` (rw-rw-r--) for files and `775` (rwxrwxr-x) for directories. This is
+  useful when using a
+  [shared development server](/doc/use-cases/shared-development-server). The
+  default permissions for cache files is system dependent.
```

Comment:
> Continues https://github.com/iterative/dvc.org/pull/2196#pullrequestreview-590249338

So are the `cache.shared` defaults always 666 and 777 in *NIX?

Whatever this is I think we need to be a bit more specific about it if we're going to mention defaults. Otherwise maybe just don't mention them.

Reply:
I think I can remove that sentence about the defaults but don't have a Windows machine to test. 😞 

In current `master`, the lines that change the permissions according to are [here](https://github.com/iterative/dvc/blob/9909ee85a1f61491a959962477208291e38f8f24/dvc/objects/db/local.py#L30)

(It's changed from the previous location and the class name is also changed.)

![Screen Shot 2021-02-18 at 14 57 45](https://user-images.githubusercontent.com/476310/108353869-b0169f00-71f9-11eb-907f-180062c7ba2e.png)

From what I see here a default `umask` of `022` will change the _default_ permissions to `644` and `755` respectively. 

When `cache.shared` is set, the permissions are **always** `664` and `775` regardless of the `umask` value. 

If you think it's better I can tell these and also mention that by default Windows doesn't provide much means to change permissions. (as @efiop said above.)

Reply:
Could you please check the changes in 53f43831ab248f1f63ea1fbe377acabb92a38712 ? @jorgeorpinel 

I can revert it and remove the default permissions discussion completely but a detailed explanation seemed better in cmd ref. Thank you.

Reply:
I need your input here @jorgeorpinel . Thank you :)

Reply:
@jorgeorpinel could we merge this now? :)

Reply:
Sorry for the delay, reviewing #2257 now...